FeatureFlags.js

```diff
@@ -1,37 +1,42 @@
+module.exports = {
+  // -----------------------------------------------------------------------------
+  // Land or remove (zero effort)
+  //
+  // Flags that can likely be deleted or landed without consequences
+  // -----------------------------------------------------------------------------
 
-export const enableComponentStackLocations = true;
+  enableComponentStackLocations: true,
 
-// -----------------------------------------------------------------------------
-// Land or remove (moderate effort)
-//
-// Flags that can be probably deleted or landed, but might require extra effort
-// like migrating internal callers or performance testing.
-// -----------------------------------------------------------------------------
+  // -----------------------------------------------------------------------------
+  // Land or remove (moderate effort)
+  //
+  // Flags that can be probably deleted or landed, but might require extra effort
+  // like migrating internal callers or performance testing.
+  // -----------------------------------------------------------------------------
 
-// -----------------------------------------------------------------------------
-// Slated for removal in the future (significant effort)
-//
-// These are experiments that didn't work out, and never shipped, but we can't
-// delete from the codebase until we migrate internal callers.
-// -----------------------------------------------------------------------------
+  // -----------------------------------------------------------------------------
+  // Slated for removal in the future (significant effort)
+  //
+  // These are experiments that didn't work out, and never shipped, but we can't
+  // delete from the codebase until we migrate internal callers.
+  // -----------------------------------------------------------------------------
 
-// -----------------------------------------------------------------------------
-// Ongoing experiments
-//
-// These are features that we're either actively exploring or are reasonably
-// likely to include in an upcoming release.
-// -----------------------------------------------------------------------------
+  // -----------------------------------------------------------------------------
+  // Ongoing experiments
+  //
+  // These are features that we're either actively exploring or are reasonably
+  // likely to include in an upcoming release.
+  // -----------------------------------------------------------------------------
 
-// -----------------------------------------------------------------------------
-// Chopping Block
-//
-// Planned feature deprecations and breaking changes. Sorted roughly in order of
-// when we plan to enable them.
-// -----------------------------------------------------------------------------
+  // Features needed for the recovery of instances that were interrupted unexpectedly (e.g. when the engine crashes and is restarted)
+  interruptedInstanceRecovery: false,
```

Comment:
I think we should stick to the Boolean naming convention of starting with a verb.
So either `recoverInterruptedInstances` or more usual for feature flags: `enableInterruptedInstanceRecovery`

Reply:
Changed the flag to the suggested `enableInterruptedInstanceRecovery` in [2156704](https://github.com/PROCEED-Labs/proceed/pull/12/commits/2156704a7b93ab6cdbe7f8d4c655f244bb7fe9dc).